### ISSUE TITLE
Rule or Function that becomes true if player has more than just one of the same item.

### ISSUE THREAD

**Issue by Taellex** [arkayenro/arkinventory#1886]
**Is your feature request related to a problem? Please describe.**
I was playing instances with a party and a ring dropped. I didn't know I had it already so I bound it to myself. I'm looking for a rule or function that categorizes items that are the same as what I'm wearing or what I have in my bags so I can't judge whether I could trade the item to another player instead of binding it to myself.

**Describe the solution you'd like**
Say I have the "Ring of Wasting Drops on me.' If another "Ring of Wasting Drops on me" drops, I would like a category to go to a section on my bags pre-selected by a rule so I would know what's going on before another player looses the chance of using the ring by me being ignorant of what I already have. Rings may be of different iLvl but same name and unique-equipped.

**Describe alternatives you've considered**
I've used the Rule Variables i.count but I think I don't understand this function.

**Additional context**
Duplicate name of an item (with varied ilvl or not); not necessarily identical items, which could also be true.

**Comment by arkayenro** [comment]
not really.  the rules apply only to the item itself, not anything else around it.

you could use the tooltip item counts to see if you have another one, or hold down the shift key to compare it with what youre wearing before equipping it.
